vyxal/context.py

```diff
@@ -0,0 +1,14 @@
+"""
+File: context.py
+Description: This is for context-related stuff.
+"""
+
+
+class Context:
```

Comment:
should this be private?

Reply:
I don't think so? Should probably consult others in the Vyxal room.

Reply:
Sure, but don't put the `ctx` variable in Context you sussy baka